uds/database/data_record/mapping_data_record.py

```
@@ -1,72 +1,103 @@
-"""Definition of TextDataRecord which is class for encode and decode values of data records."""
+"""Mapping Data Record implementation."""
 
 __all__ = ["MappingDataRecord"]
 
 from types import MappingProxyType
-from typing import Dict
+from typing import Dict, Optional, Sequence
```

Ruff (0.12.2)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)